help/destinations/catalog/email-marketing/oracle-eloqua-api.md

```diff
@@ -81,9 +85,14 @@ Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]** search for [!DNL
 
 ### Authenticate to destination {#authenticate}
 
+>[!CONTEXTUALHELP]
+>id="platform_destinations_apioracleeloqua_companyname_username"
+>title="Company name\Username"
+>abstract="Fill in this field with your company name and username from Oracle Eloqua in the form `{COMPANY_NAME}\{USERNAME}`"
+
 Fill in the required fields below. Refer to the [Gather [!DNL Oracle Eloqua] credentials](#gather-credentials) section for any guidance.
 * **[!UICONTROL Password]**: The password of your [!DNL Oracle Eloqua] account.
-* **[!UICONTROL Username]**: The username of your [!DNL Oracle Eloqua] account.
+* **[!UICONTROL Company name\Username]**: A concatenated string composed of your [!DNL Oracle Eloqua] Company Name and the [!DNL Oracle Eloqua] Username.<br>The concatenated value takes the form of `{COMPANY_NAME}\{USERNAME}`.<br> Note, do not use any braces or spaces and preserve the `\`. <br>For example if your [!DNL Oracle Eloqua] Company Name is `MyCompany` and [!DNL Oracle Eloqua] Username is `Username`, the concatenated value you will use in the **[!UICONTROL Username]** field is `MyCompany\Username`.
 
 To authenticate to the destination, select **[!UICONTROL Connect to destination]**.
 ![Platform UI screenshot showing how to authenticate.](../../assets/catalog/email-marketing/oracle-eloqua-api/authenticate-destination.png)
```

Comment:
Comment applies to row 109, right after **[!UICONTROL Description]**

I see in the latest destination configuration that the team added the new customer data field for Pod: 

```
customerDataFields: [
(+ ADDED  )     {
(+ ADDED  )       name: "pod"
(+ ADDED  )       title: "Pod"
(+ ADDED  )       description: "Pod value allocated to the user"
(+ ADDED  )       type: "string"
(+ ADDED  )       isRequired: true
(+ ADDED  )       enum: [
(+ ADDED  )         "p01"
(+ ADDED  )         "p03"
(+ ADDED  )         "p02"
(+ ADDED  )         "p04"
(+ ADDED  )         "p07"
(+ ADDED  )         "p06"
(+ ADDED  )         "p08"
(+ ADDED  )       ]
(+ ADDED  )       readOnly: false
(+ ADDED  )       hidden: false
(+ ADDED  )     }
(+ ADDED  )   ]

Can we please add **Pod** after **Description** and add a description of what it does?

And right after ### Fill in destination details {#destination-details}, leave one row blank and then please add the snippet below:

>[!CONTEXTUALHELP]
>id="platform_destinations_apioracleeloqua_pod"
>title="Pod"
>abstract="To find your pod number, log into Oracle Eloqua. Note the URL in your browser after you have logged in successfully. "
>additional-url="https://support.oracle.com/knowledge/Oracle%20Cloud/2307176_1.html" text="Oracle Knowledge base - find out your Pod number"
```

Reply:
Added, Also included a required "Security Group" pre-requisite that I found when reading up about pods. Changelog also updated.